### PR TITLE
select: doesn't open by "space" at first time (close #1486)

### LINKED ISSUE #1486: select: doesn't open by "space" at first time

### PR DESCRIPTION
**Pull request by dfilatov**
cc @narqo @veged

### CODE REVIEW

File: common.blocks/control/control.js [hunk 1]
```diff
@@ -106,7 +106,9 @@ provide(BEMDOM.decl(this.name, /** @lends control.prototype */{
 }, /** @lends control */{
     live : function() {
         this
-            .liveBindTo('control', 'focusin', this.prototype._onFocus)
+            .liveBindTo('control', 'focusin', function() {
```
Comment (narqo) [inline common.blocks/control/control.js:109]: Why do we have [separate methods in the prototype](https://github.com/bem/bem-components/blob/2e8444a4ad7bf3a859748f377749d9fb65541ae1/common.blocks/select/select.js#L221-L231) in all cases expect this one? May be we can add `_onRequestFocus` or `_onFocusin` here just to stay neat?

Comment (dfilatov) [inline common.blocks/control/control.js:109]: Because I don't want to increase a mess with managing focus state which we've already had.

Comment (narqo) [inline common.blocks/control/control.js:109]: Ok

**Comment by narqo** [comment]
:fire:

**Pull request merged**